geniza/corpus/management/commands/import_data.py

```diff
@@ -336,3 +372,153 @@ def log_creation(self, *objects):
                 object_repr=str(obj),
                 change_message=self.logentry_message,
                 action_flag=ADDITION)
+
+    def get_user(self, name, pgpid=None):
+        """Find a user account based on a provided name, using a simple cache.
+
+        If not found, tries to use first/last initials for lookup. If all else
+        fails, use the generic team account (TEAM_USERNAME).
+        """
+
+        # check the cache first
+        user = self.user_lookup.get(name)
+        if user:
+            logging.debug(
```

Comment:
Thinking about it more — script output shouldn't require you to have a logger configured a particular way to see what's going on. (I manually edited my local copy to use `stdout.write` just so I could see some of what's going on.)

I know the log level handling is nice, so suggest you either configure a script-specific logger that will output things the right way without requiring any configuration or write command methods equivalent to the log levels that check the verbosity and output the message accordingly.

Reply:
I have the same problem, but I copied this implementation from the cdhweb exodus where it works without any logging config at all, so I don't know what's wrong. afaik this shouldn't interact with django's logging settings – it's just plain python logging using the root logger. Do you have any thoughts about why it might not be working?

Reply:
pretty sure cdhweb already has logging configured and example logging set up in the sample settings.

django _uses_ standard python logging, and python logging requires you to have logging configured to see the output

Reply:
in my cdhweb localsettings, there is no logging configured at all, and the exodus logging works just fine. I tried copying some different configurations into my geniza local dev, and none had any effect on the import_data command.